docs/components/PlotRender.js

```diff
@@ -69,6 +69,9 @@ class Element {
   dispatchEvent() {
     // ignored; interaction needs real DOM
   }
+  append(...children) {
+    for (const child of children) this.appendChild(child);
```

Comment:
element.append is a little fancier than this since it also automatically promotes anything that’s not a Node to a TextNode by coercing it to a string and then wrapping it. We don’t need that functionality yet, I believe, but we should either implement it preemptively or put a comment here as a reminder.